### PR TITLE
Add Portworx as a Partner solution for external storage to Mesosphere storage docs

### PR DESCRIPTION
**Pull request by prashantsr**
Creating a separate page for Portworx on DCOS (partner integration)

## Description
https://jira.mesosphere.com/browse/DCOS-51306

## Urgency
- [ ] Blocker 
- [x] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11, 1.12).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

**Comment by ryadav88** [comment]
Internal Jira for tracking: https://jira.mesosphere.com/browse/DCOS-51306

**Comment by prashantsr** [comment]
Any updates or ETA for review? Thanks.

**Comment by SarahStegall** [comment]
I am closing this pull request because I am unable to pull from this branch. I will open another one and send you the URL.

File: pages/1.12/storage/partner integration/index.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+# Partner Integrations
```
Comment (SarahStegall) [inline pages/1.12/storage/partner integration/index.md]: WHERE IS THE METADATA?

File: pages/1.12/storage/partner integration/index.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+## Portworx
+
+Portworx solves the operational and data management problems enterprises encounter when running stateful applications on DC/OS.
```
Comment (SarahStegall) [inline pages/1.12/storage/partner integration/index.md]: This reads like an advertisement.

File: pages/1.12/storage/partner integration/index.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+* Dynamically create volumes for tasks at run time, no more submitting tickets for storage provisioning
+* Dynamically and automatically resize volumes based on demand while task is running
+* Run tasks on the same hosts that your data is located on for optimum performance
+* Avoid pinning services to particular hosts, reducing the value of automated scheduling
+* Avoid fragile block device mount/unmount operations that block or delay failover operations
+* Encrypt data at rest and in flight at the container level
+
+## Installing Portworx with DC/OS
+
+It is fairly straightforward to install Portworx with DC/OS.
```
Comment (SarahStegall) [inline pages/1.12/storage/partner integration/index.md]: No. Do not send our readers away from the DC/OS website unless absolutely necessary. Please rewrite this section to include comprehensive installation instructions, with illustrations such as the ones from the Portworx DCOS documents.

File: pages/1.12/storage/partner integration/index.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+* Avoid fragile block device mount/unmount operations that block or delay failover operations
+* Encrypt data at rest and in flight at the container level
+
+## Installing Portworx with DC/OS
+
+It is fairly straightforward to install Portworx with DC/OS. 
+For detailed instructions, visit https://docs.portworx.com/install-with-other/dcos/#installation
+
+## Building applications
+
```
Comment (SarahStegall) [inline pages/1.12/storage/partner integration/index.md]: Please add more specific instructions here. I would rewrite the ones from https://docs.portworx.com/install-with-other/dcos/application-installs/.

File: pages/1.12/storage/partner integration/index.md (hunk 1 above)
Comment (prashantsr) [reply pages/1.12/storage/partner integration/index.md]: What does this mean? Can you please explain?

File: pages/1.12/storage/partner integration/index.md (hunk 2 above)
Comment (prashantsr) [reply pages/1.12/storage/partner integration/index.md]: Fixed. Please check.

File: pages/1.12/storage/partner integration/index.md (hunk 3 above)
Comment (prashantsr) [reply pages/1.12/storage/partner integration/index.md]: It is duplication of all the information and the format won't be easy to merge. Also, it is not just one page but many such pages for Portworx and DCOS integration.

File: pages/1.12/storage/partner integration/index.md (hunk 1 above)
Comment (prashantsr) [reply pages/1.12/storage/partner integration/index.md]: Couldn't find an easy way to change the folder name. If you can do it, please do.

File: pages/1.12/storage/partner integration/index.md (hunk 4 above)
Comment (prashantsr) [reply pages/1.12/storage/partner integration/index.md]: It is duplication of all the information and the format won't be easy to merge. Also, it is not just one page but many such pages for Portworx and DCOS integration

**Comment by ck4adventure** [comment]
Closing this PR for now. If an addition of Portworx integration is needed for the docs, please file the appropriate JIRA ticket for a docs request. Also, any PR submitted MUST follow the format for our docs including folder names and the appropriate metadata at the top of the file for the docs generation.

**Pull request closed without merging**